nuxeo-justified-grid/nuxeo-justified-grid.html

```diff
@@ -119,70 +125,40 @@
         background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.4));
         text-shadow: -1px 0 rgba(0, 0, 0, 0.4), 0 1px rgba(0, 0, 0, 0.4), 1px 0 rgba(0, 0, 0, 0.4), 0 -1px rgba(0, 0, 0, 0.4);
       }
-
-      #grid .item:hover .overlay {
-        display: block;
-      }
-
-      #loading {
-        @apply --layout-vertical;
-        @apply --layout-center;
-        width: 100%;
-        display: block;
-        text-align: center;
-        padding: 32px;
-      }
-
-      #loading paper-spinner {
-        --paper-spinner-layer-1-color: var(--nuxeo-grid-selected);
-        --paper-spinner-layer-2-color: var(--nuxeo-grid-selected);
-        --paper-spinner-layer-3-color: var(--nuxeo-grid-selected);
-        --paper-spinner-layer-4-color: var(--nuxeo-grid-selected);
-      }
-
     </style>
 
     <template is="dom-if" if="[[displaySort]]">
       <nuxeo-sort-select label="[[sortLabel]]" options="[[sortOptions]]" selected="{{sortSelected}}"></nuxeo-sort-select>
     </template>
 
-    <div id="grid">
-      <template is="dom-repeat" items="[[items]]" as="item">
-        <div class="item" tabindex="0" on-tap="_tap" selected$="[[_isSelected(item, selectedItems.*)]]"
-             style$="height: [[rowHeight]]px; margin: [[margin]]px; background-image:url([[_url(item)]]);">
-          <img class="image" src="[[_url(item)]]" />
-          <paper-icon-button noink icon="icons:check" selection-mode$="[[selectionMode]]"
-                             on-tap="_check"></paper-icon-button>
-          <div class="overlay">[[item.title]]</div>
-        </div>
-      </template>
-
-      <div id="loading">
-        <paper-spinner active$="[[loading]]"></paper-spinner>
-      </div>
+    <div id="container">
+      <iron-list id="list" items="[[rows]]" as="row" on-iron-resize="_resize">
+        <template>
+          <div class="row">
+            <dom-repeat items="[[row]]">
+              <template>
+                <div class="item" tabindex="0" on-tap="_tap" selected$="[[_isSelected(item, selectedItems.*)]]"
```

Comment:
do we need tabindex="0" here ? would be cool to use the proper one if possible, you'd have to add [[tabIndex]] which is available to the template of the iron-list to the [[index]] which is part of dom-repeat model...

Reply:
tabindex="0" is needed, otherwise it's not possible to navigate between items with TAB. Using the tabindex of the iron-list will not be accurate also (because in this case it will reflect the index of the row and not the item's index.

Reply:
hence the [[tabIndex + index]] suggestion (tabindex is the row index, index is the repeat index)

Reply:
Can't make it work without tabIndex=0 so I'm keeping it as is.